.idea/codeStyles/Project.xml

```diff
@@ -42,8 +36,6 @@
     <XML>
       <option name="XML_ATTRIBUTE_WRAP" value="0" />
       <option name="XML_KEEP_BLANK_LINES" value="1" />
-      <option name="XML_ALIGN_ATTRIBUTES" value="false" />
```

Comment:
And these options?

Reply:
`3.5` changed the codestyle settings. This is automatic done by ide